src/exceptions/Bootloader-exception

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="Bootloader-exception" name="Bootloader Distribution Exception">
+      <crossRefs>
+         <crossRef>https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt</crossRef>
+      </crossRefs>
+  
+      <titleText>Bootloader Exception</titleText>
+<optional>--------------------</optional>
```

Comment:
Can we follow the [Multics precedent][1] here and allow different numbers of hyphens?  Previous discussion [here][2].

And this underline is really title markup, so:

* I'd prefer it to go inside `<titleText>`.
* I'd rather not include it in the canonical HTML at all.

Putting all of those suggestions together gives something like:

```xml
<titleText>
  <p>Bootloader Exception<alt match="-*" name="titleUnderline"/></p>
</titleText>
```

We don't currently distinguish between “canonical HTML content” (which should not have the underline) and “canonical text/plain content” (which maybe should have the underline), but I think not having the underline in text/plain generated from our XML is a lesser evil than having underscores in HTML generated from our XML.

[1]: https://github.com/spdx/license-list-XML/blob/52f17985f621530754add25316605748d9f19c82/src/Multics.xml#L33-L35
[2]: https://github.com/spdx/license-list-XML/pull/474#discussion_r148378189

Reply:
fine. let's make consistent with Multics then.